http/java/okhttp/lib/src/main/java/com/microsoft/kiota/http/OkHttpRequestAdapter.java

```diff
@@ -313,7 +313,7 @@ private Request getRequestFromRequestInformation(@Nonnull final RequestInformati
                                 new RequestBody() {
                                     @Override
                                     public MediaType contentType() {
-                                        final String contentType = requestInfo.headers.containsKey(contentTypeHeaderKey) ? requestInfo.headers.get(contentTypeHeaderKey) : "";
+                                        final String contentType = requestInfo.getRequestHeaders().containsKey(contentTypeHeaderKey) ? requestInfo.getRequestHeaders().get(contentTypeHeaderKey) : "";
```

Comment:
call `getRequestHeaders()` once before this line and assign into a variable to avoid calling `clone()` twice.